pkg/utils/validation/admissionplugins/admissionplugins.go

```diff
@@ -53,8 +53,6 @@ var (
 		"DefaultIngressClass":                  {AddedInVersion: "1.18"},
 		"DefaultStorageClass":                  {},
 		"DefaultTolerationSeconds":             {},
-		"DenyEscalatingExec":                   {RemovedInVersion: "1.21"},
```

Comment:
I think we can also remove all `AddedInVersion` fields if it's <= 1.22.

Reply:
Yes, but we don't gain much. I think we can keep it.